hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -68,7 +68,7 @@
   public static final HoodieFileFormat DEFAULT_LOG_FILE_FORMAT = HoodieFileFormat.HOODIE_LOG;
   public static final String DEFAULT_PAYLOAD_CLASS = OverwriteWithLatestAvroPayload.class.getName();
   public static final Integer DEFAULT_TIMELINE_LAYOUT_VERSION = TimelineLayoutVersion.VERSION_0;
-  public static final String DEFAULT_ARCHIVELOG_FOLDER = "";
+  public static final String DEFAULT_ARCHIVELOG_FOLDER = "archived";
```

Comment:
this is already overridden at the datasource and deltastreamer level.. Please revert this since it can affect all the old tables written into `.` ..

@n3nash can you advise here.. only reason to not change is all the old data at uber..

Reply:
@vinothchandar that's correct, we don't override this and all commits archived data at Uber is written to .hoodie/`.`

@hddong this change doesn't seem necessary but has serious concerns for older data written (even by other users who might not be overriding the archived folder name), could you please revert it ?